dts/arm/st/f3/stm32f302.dtsi

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2017 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <st/f3/stm32f3.dtsi>
+
+/ {
+	soc {
+		i2c2: i2c@40005800 {
+			compatible = "st,stm32-i2c-v2";
+			clock-frequency = <I2C_BITRATE_STANDARD>;
+			#address-cells = <1>;
+			#size-cells = <0>;
+			reg = <0x40005800 0x400>;
+			clocks = <&rcc STM32_CLOCK_BUS_APB1 0x00400000>;
+			interrupts = <33 0>, <34 0>;
+			interrupt-names = "event", "error";
+			status = "disabled";
+			label= "I2C_2";
+		};
+
```

Comment:
I just missed it.  I will add it.

Regarding how the f3 family device tree is organized, I have ideas on how it might be better organized.  Should I use the developer mailing list or open an issue in order to discuss this?

Reply:
Please raise an issue. Proposition should apply to all stm32 series.